api_server.py

```
@@ -561,4 +564,4 @@ def add_security_headers(response):
 """
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))
+    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))  # nosec B104
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)